File: foundation 3 month/lesson9/homework/ex_2.py
import random
def check(ms):
	try:
		print(10/len(ms))
	except:
		print("Soni norga bulish mumkin emas: ")


k=int(input("K ni kiriting: "))
arr=list()
for x in range(k):
	arr.append(random.randint(10,30))
print(arr)
check(arr)

File: foundation/foundation 2 month/p_exam/real/ex_2.c
#include <stdio.h>
#include <stdlib.h>
double   check(int k)
{
	return 3.14*k*k;
}
int main()
{
	int n;
	printf("Kiriting: ");
	scanf("%d",&n);
	double d = check(n);
	printf("%.2lf",d);
}
